packages/jellyfish-api-core/__tests__/category/masternode/listMasternodes.test.ts

```diff
@@ -0,0 +1,51 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { MasternodePagination } from '../../../src/category/masternode'
+
+describe('Masternode', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  it('should list masternodes', async () => {
+    const masternodes = await client.masternode.listMasternodes()
+
+    for (const masternode in masternodes) {
```

Comment:
Same here, empty masternode array still pass test.

Reply:
Added `    expect(Object.keys(masternodeTransaction).length).toStrictEqual(1)
` and `    expect(Object.keys(masternodeTransaction).length).toBeGreaterThanOrEqual(1)
` respectively.